eras/conway/impl/CHANGELOG.md

```diff
@@ -2,11 +2,12 @@
 
 ## 1.20.0.0
 
-- Move some hard-fork triggers and export them from `Cardano.Ledger.Conway` module.
```

Comment:
@aniketd I would like to ask you to not use `-` in changelog for bullet points anymore.
I ended up doing the same fix in my PR.